library/annotate/annotate.py

```diff
@@ -0,0 +1,38 @@
+# MIT LINCENCE. 2021
+#
+# This file is part of an academic capstone project,
+# and it is made for AMD as part of efforts to automate
+# the open source ROCM math libraries performance analytics.
+# Contact The AMD rocm team for use and improvements on the project.
+# The team: Victor Tuah Kumi, Aidan Forester, Javier Vite, Ahmed Iqbal
+# Reach Victor Tuah Kumi on LinkedIn
+
```

Comment:
Please everyone add your contact info or profile where you want to be reached at.